Cython/Compiler/Main.py

```diff
@@ -735,6 +735,9 @@ def compile_multiple(sources, options):
     a CompilationResultSet. Performs timestamp checking and/or recursion
     if these are specified in the options.
     """
+    if len(sources) > 1 and options.module_name:
```

Comment:
I'd reverse this. The issue is not that there are multiple sources, but that the module name was passed, which then dictates that there can only be one source. (Also, it's probably less likely for users to pass a module name than multiple sources.)
```suggestion
    if options.module_name and len(sources) > 1:
```